nilearn/glm/first_level/first_level.py

```diff
@@ -694,7 +694,7 @@ def _create_single_design(self, n_scans, events, confounds, run_idx):
             self.drift_model,
             self.high_pass,
             self.drift_order,
-            self.fir_delays,
+            self.fir_delays_,
```

Comment:
The same here.

Reply:
same here: will update sklearn_is_fitted to check for its presence